src/Equinor.ProCoSys.IPO.Command/InvitationCommands/CancelPunchOut/CancelPunchOutCommandHandler.cs

```diff
@@ -79,13 +79,19 @@ private async Task CancelFusionMeetingAsync(Guid meetingId)
             {
                 if (e.Code.ToString().ToUpperInvariant() == "FORBIDDEN")
                 {
-                    _logger.LogError(e, $"Unable to cancel outlook meeting for IPO.");
+                    _logger.LogError($"Unable to cancel outlook meeting for IPO. Inconsistency between Fusion and IPO regarding rights for meeting with id {meetingId}. Stack trace: {e.StackTrace}");
                 }
                 else
                 {
                     throw;
                 }
             }
+            catch (ArgumentException e)
+            {
+                _logger.LogWarning($"Unable to cancel outlook meeting in Fusion IPO with id {meetingId}. " +
```

Comment:
Curious, what is the scenario where we end up in this catch clause?

Reply:
Fusion has been unstable at IPO creation time. Hence no meeting registered in Fusion. In such cases Fusion throws ArgumentException.